source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -38,13 +38,32 @@ OpenGLContext: class extends GpuContext {
 	_pointsShader: OpenGLMap
 	_meshShader: OpenGLMapMesh
 	_renderer: GLRenderer
-	_recycleBin := _RecycleBin new()
+	_recycleBinMonochrome: RecycleBin<OpenGLMonochrome>
```

Comment:
That's not an issue at the moment, that's why I feel it's a premature optimization. I think we can go even further and begin with a single bin and optimize it when we need to. It would make the code a lot cleaner.